.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -6,50 +6,49 @@ week: 1
 step: 4
 name: Week 1 Step 4
 ---
+Week 1 Step 4 ⬤⬤⬤⬤◯◯◯◯ | 🕐 Estimated completion: 35mins-45 mins
+### Writing Your First Serverless Function
+It's Time! to write your first Serverless function. 
 
-## Week 1 Step 4
 
-🕐 Estimated completion: 35-45mins
+### ✅  Task:
+- [ ]  run `git pull` 
+- [ ]  make sure you are in the `week1` branch 
+- [ ]  place your function URL in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `HTTP_ENDPOINT`
```

Comment:
This can be a single task: locally clone the repository. The steps can be rewritten as instructions, or they can be left as hints. Either way, this task **should be in its own issue**